enaml/core/parser/lexer3.py

```diff
@@ -81,7 +81,7 @@ def make_token_stream(self):
 
         """
         token_stream = super(Python35EnamlLexer, self).make_token_stream()
-        token_stream = self.analyse_async(token_stream)
+        #token_stream = self.analyse_async(token_stream)
```

Comment:
The idea with this pass was to preserve the behavior of python that allows async and await as name in non async functions. I agree this is not critical. I will try to see if we can fix it. I would prefer to stay as close as possible to the promise that any valid Python is valid enaml.

Reply:
Ok, yeah that makes sense. It should be left in for 3.5 and 3.6 but overridden to ignore it for 3.7 (in python 3.7 async and await are reserved keywords).

Some more tests should be added to check this then.

Reply:
You need to reactivate this for the new tests to pass.